sdk/webpubsub/Azure.Messaging.WebPubSub/src/Generated/HealthApiClient.cs

```diff
@@ -57,6 +57,7 @@ public HealthApiClient(string endpoint, WebPubSubServiceClientOptions options)
         /// <param name="context"> The request context, which can override default behaviors of the client pipeline on a per-call basis. </param>
         /// <exception cref="RequestFailedException"> Service returned a non-success status code. </exception>
         /// <returns> The response returned from the service. </returns>
+        /// <remarks> Get service health status. </remarks>
```

Comment:
Here is an instance of a public API where `remarks` is being added because of duplicated text.  I agree with @jsquire and @AlexanderSher that if the text is identical, there is no reason to add the `remarks` block and no loss of information.